testsuite/features/min_virt_kvm_guests.feature

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2018 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Be able to manage KVM virtual machines via the GUI
+
+@virtualization_kvm
+  Scenario: Bootstrap virtual host
+    Given I am authorized
+    When I go to the bootstrapping page
+    Then I should see a "Bootstrap Minions" text
+    When I enter the hostname of "kvm-server" as hostname
+    And I enter "22" as "port"
+    And I enter "root" as "user"
+    And I enter "kvm-server" password
+    And I select "1-SUSE-PKG-x86_64" from "activationKeys"
+    And I select the hostname of the proxy from "proxies"
+    And I click on "Bootstrap"
+    And I wait until I see "Successfully bootstrapped host! " text
+    And I navigate to "rhn/systems/Overview.do" page
+    And I wait until I see the name of "kvm-server", refreshing the page
+    And I wait until onboarding is completed for "kvm-server"
+    # Shorten the virtpoller interval to avoid loosing time
+    And I reduce virtpoller run interval on "kvm-server"
+
+@virtualization_kvm
+  Scenario: Setting the virtualization entitlement
+    Given I am on the Systems overview page of this "kvm-server"
+    When I follow "Details" in the content area
+    And I follow "Properties" in the content area
+    And I check "virtualization_host"
+    And I click on "Update Properties"
+    Then I should see a "Since you added a Virtualization system type to the system we also performed some extra steps to ensure your system will be able to manage virtual guests better." text
```

Comment:
I would add a comma before " we" (in the software too).

Would it be bad to test only for a part of the message you test? For example "your system will be able to manage virtual guests" :smiley:

Reply:
no that wouldn't be bad since there are no chances to match another message